dra-evolution/testdata/pod-one-container-one-gpu-one-vf.yaml

```diff
@@ -6,29 +6,58 @@
 # just be ignored for that purpose.
 
 apiVersion: resource.k8s.io/v1alpha2
-kind: ResourceClaimTemplate
+kind: ResourceClaim
 metadata:
   name: foozer-plus-vf
   namespace: default
+  finalizers:
+  - dra.k8s.io/delete-protection
 spec:
-  spec:
+  requirements:
+  - match:
+      attribute: pcie-root.dra.k8s.io
+  requests:
+  - classes:
+    - name: example.com-foozer
+  - classes:
+    - name: example.org-sriov-nic
     requirements:
-    - match:
-        attribute: pcie-root.dra.k8s.io
-    requests:
-    - classes:
-      - name: example.com-foozer
-    - classes:
-      - name: example.org-sriov-nic
-      requirements:
-      - device:
-          selector: "device.stringAttributes['sriovType.example.org-sriov-nic'] == 'vf'"
+    - device:
+        selector: "device.stringAttributes['sriovType.example.org-sriov-nic'] == 'vf'"
+status:
+  allocation:
+    availableOnNodes:
+      nodeSelectorTerms:
+      - matchFields:
+        - key: name
+          operator: in
+          values:
+          - worker-1
+    shareable: true
+    driverData:
+    - driverName: example.com-foozer
+      structured:
+        nodeName: worker-1
+        results:
+        - device:
+            name: gpu-1
+    - driverName: example.org-sriov
+      structured:
+        nodeName: worker-1
+        results:
+        - device:
+            name: nic-1
+  reservedFor:
+  - resource: pods
+    name: foozer
+    uid: abcd-1234
```

Comment:
Were the changes in this file meant to be part of this commit?

Reply:
Yes. I included this in c920b2be9077bdb0d0b because I wanted to show what a non-trivial status would look like.

I could have added the "generated" ResourceClaim at the bottom instead of converting the ResourceClaimTemplate to a ResourceClaim, but that felt a bit redundant.

Reply:
It might be better to create a separate example for this. It's now hard to look at this file and get at the core of what it is meant to demonstrate, i.e. one container asking for one GPU and one vf.

Reply:
Ack.